rust/kernel/print.rs

```diff
@@ -439,3 +439,31 @@ macro_rules! pr_cont (
         $crate::print_macro!($crate::print::format_strings::CONT, true, $($arg)*)
     )
 );
+
+/// Equivalent to [`std::dbg`](https://doc.rust-lang.org/std/macro.dbg.html), but uses `pr_debug`.
+#[macro_export]
+macro_rules! dbg {
+    // Comment from original / std implementation:
+
```

Comment:
If we move to another file, and the top-level comment explains where the code comes from, then this can be removed.